.github/workflows/iroha2-pr-heavy.yml

```diff
@@ -1,72 +1,63 @@
-name: I2::Dev::Tests (Heavy)
+name: I2::Release::Tests
 
 on:
   pull_request:
     branches: [iroha2]
-    tags:
-      - "v*.*.*"
-    paths:
-      - "**.rs"
-      - "**.json"
-      - "**.toml"
-      - "**.yml"
-
-      # Not part of the workspace
-      - "!wasm/**"
 
 env:
   CARGO_TERM_COLOR: always
 
 jobs:
-  test-docker:
-    runs-on: ubuntu-latest
+  cli-test:
+    runs-on: [self-hosted, linux]
     container:
       image: 7272721/i2-ci:latest
```

Comment:
I'm not sure, but why do we need fake (according to its name) dockerhub account for this? Can we have normal account or store Dockerfile in repository to build image from scratch for the first time?

Reply:
It's not fake. It's actually a Dockerhub account that stores our custom CI image, and only that image. It's so we don't have to download rust and install a bunch of `apt` packages every time, and instead use the cached image from inside the pipeline. See #1525.